tendermint/src/node/info.rs

```diff
@@ -58,10 +58,15 @@ pub struct ProtocolVersionInfo {
 }
 
 /// Listen address information
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize, Default)]
```

Comment:
It seems like this could have a more useful default (like, say, `0.0.0.0`, or whatever Tendermint does)

Reply:
Agree, I think the default is "tcp://127.0.0.1:26656";

Reply:
But, again, a concret network address might not suit for `Default` trait. Maybe I remove this instance.